### ISSUE TITLE
Pushable feature - min range can be skipped with axis click

### ISSUE THREAD

**Issue by tasiek** [react-component/slider#200]
Pushable's range can be "cheated" with clicking on axis. I suppose it's not a desirable behaviour.

![rc-slider-range-pushable-axis-click](https://cloud.githubusercontent.com/assets/4588259/21640944/cfe6f480-d278-11e6-8712-ddcf4a641c0a.gif)

In this example, pushable's range is set to 250.

**Comment by paranoidjk** [comment]
@tasiek do you mean you set `props.step=250` ?

all range demo can be click to trigger, see this http://react-component.github.io/slider/examples/range.html

i think it's a useful feature

**Comment by tasiek** [comment]
@paranoidjk 
No, i mean setting `pushable` to a number (250, in my case). Then 250 should be a minimal possible range, and i'm able to omit that with clicking.
I was checking the code and there's no pushable feature checked when clicking on axis (probably `onStart` method)
